butterknife/src/main/java/butterknife/ButterKnife.java

```diff
@@ -85,7 +85,7 @@ private ButterKnife() {
 
   /** An unbinder contract that can be bind with {@link butterknife.Unbinder}. */
   @SuppressWarnings("unused") // Used by generated code.
-  public interface Unbinder {
+  public interface ViewUnbinder<T> {
```

Comment:
Sorry for squeezing this in... but `@Unbinder ButterKnife.Unbinder` is driving me crazy...